server/models/helpers/ProsemirrorHelper.tsx

```diff
@@ -155,6 +169,67 @@ export class ProsemirrorHelper {
     return mentions;
   }
 
+  static getNodeForMentionEmail(doc: Node, mention: MentionAttrs) {
+    let blockNode: Node | undefined;
+    const potentialBlockNodes = ["table", "checkbox_list", "paragraph"];
```

Comment:
We discussed a table row being the right point to cut, tables can be quite large so just wondering on the `table` inclusion here

Reply:
Oh yes, row is still the cut point.
We need the table as it has the necessary styles.

On L217, a new parent node gets created with a single child that has the mention.